### PR TITLE
Fix to compile in newest devkitarm

### PR DESCRIPTION
**Pull request by burhanr13**

### CODE REVIEW

File: source/main.cpp [hunk 1]
```diff
@@ -10,65 +10,36 @@
...
-    nullptr
-};
+    "file",  0,       nullptr, nullptr, file_write, nullptr, nullptr,
+    nullptr, nullptr, nullptr, nullptr, nullptr,    nullptr, nullptr,
+    0,       nullptr, nullptr, nullptr, nullptr,    nullptr, nullptr,
+    nullptr, nullptr, nullptr, nullptr, nullptr};
 
-static bool TryOpeningNewFile(const char* path)
-{
+static bool TryOpeningNewFile(const char* path) {
```
Comment (lioncash) [inline source/main.cpp]: Seems like there's a bunch of unrelated style changes

File: source/main.cpp [hunk 2]
```diff
@@ -256,12 +196,11 @@ int main(int argc, char** argv)
     }
 
     // Exit services
-    sdmcExit();
+    // sdmcExit();
```
Comment (lioncash) [inline source/main.cpp]: Perhaps just remove this outright if it's no longer necessary?

**Review by lioncash — COMMENTED** [review]
Just a few minor nits, but I appreciate the patch!

File: source/main.cpp (hunk 1 above)
Comment (burhanr13) [reply source/main.cpp]: sorry, looks like i had the autoformatter on. Have fixed it

**Review by lioncash — APPROVED** [review]
looks good, thanks!
